data/meetup/nizhny-novgorod-russia.md

```diff
@@ -0,0 +1,10 @@
+---
+location: 'Nizhny Novgorod, Russia'
+url: 'https://emberjs-nn.timepad.ru/events/'
+lat: 56.319827
+lng: 44.001361
+organizers:
+  - organizer: Natalia Fel
```

Comment:
On Twitter the organizer's name is mentioned as Natasha Fel. Could you please tag her on this review so that she can confirm this?

Reply:
This is basically the same name, but she prefers the form "Natasha". Thanks for noticing this! Fixed.